src/pages/index.js

```diff
@@ -93,34 +117,120 @@ function setFormInfo(nameSelector, detailsSelector) {
 }
 
 function createCard(data) {
-  const cardElement = new Card({ data, handleImageClick }, "#cards-template");
+  const cardElement = new Card(
+    { data, handleImageClick },
+    "#cards-template",
+    deleteCardConfirm,
+    handleCardLike
+  );
   return cardElement.generateCard();
 }
 
+function deleteCardConfirm(id, cardElement) {
+  deleteCardConfirmModal.open();
+  deleteCardConfirmModal.setSubmitAction(() => {
+    handleDeleteCard(id, cardElement);
+  });
+}
+
 function handleImageClick(imgData) {
   previewModal.open(imgData);
 }
 
+function handleCardLike(id, likeFunction, unlikeFunction, button, card) {
+  if (!card._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.